src/EditorFeatures/CSharp/CSharpEditorResources.resx

```diff
@@ -182,4 +182,256 @@
     <value>Add Missing Usings on Paste</value>
     <comment>"usings" is a language specific term and should not be localized</comment>
   </data>
+  <data name="Avoid_expression_statements_that_implicitly_ignore_value" xml:space="preserve">
+    <value>Avoid expression statements that implicitly ignore value</value>
```

Comment:
1.  did tehse values get moved from somewhere else?  are they copies?
2. isn't this a value that would be shared between C# and VB?  why is it in CSharpEditorResources?

Reply:
> did tehse values get moved from somewhere else? are they copies?

These values are copied from a higher layer. There is no reason these types need to be in the VS layer instead of the Editor layer other than the resources but I can move the types back if we don't want to worry about it.

> isn't this a value that would be shared between C# and VB? why is it in CSharpEditorResources?

`CSharpCodeStyleOptions.UnusedValueExpressionStatement` is a C# only option as far as I can tell. if we ever offer the ability to enforce this for VB then we should move it.